### PR TITLE
Add exporter for Alibaba Cloud

### PR DESCRIPTION
**Pull request by aylei**
Add exporter for Alibaba Cloud, default port has been signed in [Default port allocations](https://github.com/prometheus/prometheus/wiki/Default-port-allocations) as 9525

Signed-off-by: alei <rayingecho@gmail.com>

### CODE REVIEW

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
 
 ### Other monitoring systems
    * [Akamai Cloudmonitor exporter](https://github.com/ExpressenAB/cloudmonitor_exporter)
+   * [Alibaba Cloud Exporter](https://github.com/aylei/aliyun-exporter)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: This looks to be a Cloudmonitor exporter, rather than Alibaba generally

Comment (aylei) [reply content/docs/instrumenting/exporters.md]: How about "Alibaba Cloudmonitor exporter"?

Comment (brian-brazil) [reply content/docs/instrumenting/exporters.md]: Sure

**Comment by brian-brazil** [comment]
Thanks!

**Comment by aylei** [comment]
@brian-brazil Thanks for your review!

**Pull request merged**